data/campaigns/The_Rise_Of_Wesnoth/scenarios/16_The_Kalian.cfg

```diff
@@ -42,11 +42,10 @@
 
     [side]
         side=3
-        type=Elvish Champion
+        type=Elvish Champion Kalian
```

Comment:
why this change?

Reply:
hmm ok. But having such a custom unit will make it incompatible with older clients, (since the unit_type data is not sended over network, if the newer client is host, older cleints will get a 'unknown unit type' error).  So an [object] based solution would really be preferrable if possible.

Reply:
I guess `[effect]apply_to=image_mod` could work here by somehow blanking out the standard image and blitting on the new image, but that's extremely ugly... I think it would be the only way to get the effect with an `[object]`.